lua/gitblame/init.lua

```diff
@@ -293,7 +296,7 @@ local function get_blame_text(filepath, info, callback)
         and info.author ~= "External file (--contents)"
         and info.author ~= "Not Committed Yet"
 
-    if is_info_commit then
+    if info and is_info_commit then
```

Comment:
Even though we check `info` above luals still thinks `info` could be `nil` here so I just added it here again :shrug: